src/helpers/helper.js

```diff
@@ -69,6 +69,18 @@ const helper = {
   isContentJson(res) {
     const type = res && res.headers['content-type'];
     return type ? type.includes('application/json') : false;
+  },
+
+  getStashKey(key) {
+    return `$S{${key}}`;
+  },
+
+  getMapKey(key) {
+    return `$M{${key}}`;
+  },
+
+  getFunctionKey(key) {
+    return `$F{${key}}`;
   }
 
```

Comment:
Let's move this functions to `exports/stash.js`. Placing these functions over there would be more relevant. And also please update `stash.d.ts` file.